examples/fluids/meshes/cylinder.geo

```diff
@@ -7,7 +7,10 @@ Ny  = N/2 + 1; Ry = 2.00;
 Nb  = N/2 + 1; Rb = 0.9;
 Nc  = N/2 + 1; Rc = 1.00;
 
-zspan = 0.2;
+DefineConstant[
+  zspan = {0.2, Min .1, Max 0.2, Step .1,
+    Name "Parameters/Zspan"}
+];
```

Comment:
Let's allow it to get big so we can have multiple layers in the z direction, for a more genuinely 3D problem that transitions to turbulence at lower Reynolds number than the 2D problem.

Reply:
How big should we get it?

Reply:
You can leave the default, but make `Max 10` (or interpret it as up to 10 times the cylinder diameter). Then populate with however many layers are needed for the elements to have roughly equal aspect to the wake region.